tests/UI/specs/Login_spec.js

```diff
@@ -49,8 +49,6 @@ describe("Login", function () {
 
     it("should redirect to login when logout link clicked", function (done) {
         expect.screenshot("login_form").to.be.capture("logout_form", function (page) {
-            page.click("#topBars span.title:contains(superUserLogin)");
```

Comment:
Did not work before. This was code from when the username opened a dropdown menu